source/cloud/azure/azureml.md

```diff
@@ -142,17 +151,11 @@ You can define an environment from a [pre-built](https://learn.microsoft.com/en-
 Create your custom RAPIDS docker image using the example below, making sure to install additional packages needed for your workflows.
 
 ```dockerfile
-
 # Use latest rapids image with the necessary dependencies
 FROM {{ rapids_container }}
 
-# Update and/or  install required packages
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends build-essential fuse && \
-    rm -rf /var/lib/apt/lists/*
-
-# Activate rapids conda environment
-RUN /bin/bash -c "source activate rapids && pip install azureml-mlflow"
+RUN conda install --yes -c conda-forge 'dask-ml>=2024.4.4' \
+ && pip install azureml-mlflow
```

Comment:
Unfortunately `azureml-mlflow` is not available as a conda package. Tracking request to get it on `conda-forge`, if you want to subscribe: https://github.com/conda-forge/staged-recipes/issues/23432